app/jobs/handover_follow_up_job.rb

```diff
@@ -0,0 +1,32 @@
+class HandoverFollowUpJob < ApplicationJob
+  queue_as :default
+
+  def perform(date)
+    ldus = LocalDivisionalUnit.all.where.not(email_address: nil)
```

Comment:
There is a scope for this which does something slightly different - LocalDivisionalUnit.without_email_addresses

Reply:
I originally did that but it was quite inefficient